### ISSUE TITLE
Can I connect two graphs and still retain their individual references?

### ISSUE THREAD

**Issue by vabichequer** [paulbrodersen/netgraph#40]
Hi! I am using Networkx to draw several graphs in a figure. Each graph is represented in a circular layout and may have an edge with another circle. The issue is that I can't use Networkx's union function, otherwise I'll end up with a single graph if two individual ones have an edge between them. What I wanted was to just join them with an edge but still retain their individual references, since it may be that I there is a link between one of the graphs mentiones earlier and another one in the future. The reference is important to me because I still want to be able to plot them individually in a circle layout, rather than plotting them together in a circle layout.

Cheers!

**Comment by paulbrodersen** [comment]
>  I am using Networkx to draw several graphs in a figure.

1. Are you using networkx or netgraph for plotting?
2. If you are using netgraph, [please create a minimal, working example](https://stackoverflow.com/help/minimal-reproducible-example) that demonstrates the current output. Also, add a sketch with your desired output (easiest done by drawing on paper and taking a picture with your phone).  At the moment, it is unclear what the issue is.

**Comment by vabichequer** [comment]
Sure. Sorry, I didn't detail it enough. I am using networkx still, but I'm thinking on migrating to netgraph, if I can do what I'm asking. 

This image [https://imgur.com/a/RTAO7vE](url) shows what I'm trying to achieve.

The main thing is that I can't create an edge between the two graphs without turning them into only one. The only way to do this is by using the union function from networkx, which returns a new, relabeled graph, with the edge I want, but with the two graphs mixed together. 

What I want to know is if netgraph is able to create edges between graphs without mixing them together. That is, if I'm able to create an edge between two graphs and still plot them separately.

I can't create a minimal working example, I'm sorry. This is more of a feature question, I guess.

**Comment by vabichequer** [comment]
Not really. Although it seems like it, I actually want to draw an arrow between two nodes from different graphs. I figured that I could use the positional argument from the circular_layout function in networkx and use [ConnectionPatch](https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.patches.ConnectionPatch.html) to make the connection between two subplots. I just didn't want to go through the hassle of decoding the positions. I thought that maybe networkx or netgraph had a function such as "edge between graphs" that would accept two graphs and two indexes and link them, but that doesn't seem to be the case. 

Since I figured a solution, I guess we could close the issue.
